src/api/helpers/get-unverified-accounts/index.ts

```diff
@@ -0,0 +1,41 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import { Account } from '../../types';
+
+/**
+ * getUnverifiedAccounts
+ * gets unverified accounts - not updated for 24hours
+ * returns array of accounts including status
+ * @param {Context} context
+ * @returns {Promise<Account[]>} Array of accounts
+ */
+const getUnverifiedAccounts = async (context: Context): Promise<Account[]> => {
+  try {
+    console.info('Getting unverified accounts - getUnverifiedAccounts helper');
+
+    /**
+     * queries accounts which have a verificationExpiry before now
+     * and where their isVerified, isBlocked or isInactive statuses are false
+     * returns array accounts and their status
+     */
+    const accounts = (await context.query.Account.findMany({
+      where: {
+        AND: [
+          { verificationExpiry: { lt: new Date() } },
+          { status: { isVerified: { equals: false } } },
+          { status: { isBlocked: { equals: false } } },
+          { status: { isInactive: { equals: false } } },
+        ],
+      },
```

Comment:
If the function is called `getUnverifiedAccounts`, why does it care about `isBlocked` and `isInactive`?

Reply:
If an account is already inActive, then it should not be returned as will waste resources.  And also I thought if an account is blocked, then should not also be set to isInactive

Reply:
`isVerified` and `isInactive` makes sense, I don't think `isBlocked` should be included here.

A user could become blocked by trying to sign in a few times, and then abandon trying to sign in. CRON job could handle this

Reply:
Not a blocker - can be done in the next PR, could we add some documentation about why we check that `isVerified` is false?